.gitignore

```diff
@@ -1,36 +1,18 @@
-node_modules/
+
+# IDE
+/.idea
+/.awcache
+/.vscode/*.code-workspace
+
```

Comment:
Shouldn't we add `/.vscode/settings.json` too?

Reply:
yes you are right, i have fixed it